test/e2e/config/azure-dev.yaml

```diff
@@ -169,6 +180,7 @@ variables:
   AKS_KUBERNETES_VERSION_UPGRADE_FROM: "latest-1"
   KUBERNETES_VERSION: "${KUBERNETES_VERSION:-stable-1.27}"
   FLATCAR_KUBERNETES_VERSION: "${FLATCAR_KUBERNETES_VERSION:-stable-1.26}"
+  API_UPGRADE_TEST_KUBERNETES_VERSION: "v1.26.12"
```

Comment:
I think we do want the "FROM" in here somewhere so we know that this version is used in an upgrade test as the _original_ version we build a cluster from, before we upgrade to. Maybe

`KUBERNETES_VERSION_API_UPGRADE_FROM`

Sorry for all the bikeshedding here. :/

Also, curious why we don't use `stable-1.26` instead of a specific version?

Reply:
No problem! It's my bad for not getting it right. 

> Also, curious why we don't use stable-1.26 instead of a specific version?

I'm not too sure as that is what was in the test before. This was added by @CecileRobertMichon in #4433 in case she knows the answer.